lib/registry.js

```diff
@@ -57,19 +58,18 @@ class Registry {
 				? `{${formattedLabels.join(',')}}`
 				: '';
 
-			values.push(
-				`${metricName}${labelsString} ${getValueAsString(val.value)}`,
-			);
+			fullMetricLine = `${metricName}${labelsString} ${getValueAsString(
+				val.value,
+			)}`;
```

Comment:
nit: why did you declare it above? Should be fine to just `let fullMetricLine = ...` here.